src/tree/AttachedAccountsTreeItem.ts

```diff
@@ -107,12 +111,24 @@ export class AttachedAccountsTreeItem extends AzureParentTreeItem {
 
         const attachedAccounts: AzureTreeItem[] = await this.getAttachedAccounts();
 
-        return attachedAccounts.length > 0 ? attachedAccounts : [new GenericTreeItem(this, {
-            contextValue: 'cosmosDBAttachDatabaseAccount',
-            label: 'Attach Database Account...',
-            commandId: 'cosmosDB.attachDatabaseAccount',
-            includeInTreeItemPicker: true
-        })];
+        if (attachedAccounts.length > 0) {
+            return attachedAccounts;
+        } else {
+            const genericTreeItemWithoutEmulator = new GenericTreeItem(this, {
```

Comment:
Could you update the `genericTreeItemWithoutEmulator` and `genericTreeItemWithEmulator` variable names? We show `genericTreeItemWithoutEmulator` whether or not the emulator is enabled.

Maybe something like `attachDatabaseAccount` and `attachEmulator` could be more fitting